src/ui/inspector/PlotElementsPool.vue

```diff
@@ -0,0 +1,333 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2022, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+<template>
+<div class="c-elements-pool">
+    <Search
+        class="c-elements-pool__search"
+        :value="currentSearch"
+        @input="applySearch"
+        @clear="applySearch"
+    />
+    <div
+        class="c-elements-pool__elements"
+    >
+        <ul
+            v-if="yAxis1.length > 0
+                || yAxis2.length > 0
+                || yAxis3.length > 0"
+            id="inspector-elements-tree"
+            class="c-tree c-elements-pool__tree"
+        >
+            <element-item-group
+                :parent-object="parentObject"
+                :allow-drop="allowDrop"
+                label="Y Axis 1"
+                @drop-group="moveTo($event, 0, Y_AXIS_1)"
```

Comment:
Remove the hardcoded Y_AXIS_* ids. Instead, define a generic yAxes array that is a list of objects with the y-axis id and elements to show in that y axis. You can get the list of yAxes from the parentObject configuration like this:
```
setYAxisIds() {
            const configId = this.openmct.objects.makeKeyString(this.parentObject.identifier);
            this.config = configStore.get(configId);
            this.yAxes.push({
                id: this.config.yAxis.id,
                elements: []
            });
            if (this.config.additionalYAxes) {
                this.config.additionalYAxes.forEach(yAxis => {
                    this.yAxes.push({
                        id: yAxis.id,
                        elements: []
                    });
                });
            }
        }
```